### ISSUE TITLE
[Feature][N64] Overclock / upscale support

### ISSUE THREAD

**Issue by JoeMatt** [Provenance-Emu/Provenance#1957]
Mupen has overclocking options I haven't looked into

There used to be a parameter "mupen64-virefresh = "2200""
but that's renamed or moved.

**Comment by jpike88** [comment]
From my research you'd need a virefresh setting per game, and the mupen64plus team tried initially to define that but ended up giving up (https://github.com/mupen64plus/mupen64plus-core/pull/204/files).

As for upscaling, it seems you'd need parallel-rdp but that's only available on mupen64plus-libretro-nx, which I believe isn't the core you've chosen to run with

**Comment by JoeMatt** [comment]
Right, that core didn't even exist when we added Mupen, at least not with GLES/ARM support.

That being said, I do have a test core as a part of my libretro wrapper feature coming Q1 2023.

**Comment by JoeMatt** [comment]
@claude we have a mupen update ticket #1632 #998 so maybe we can add this to our mupen, or mabye the current version we already have?

tag the info on those tickets so we're aware of it when working on mupen, if it's something we can just do now, make a new work ticket

**Comment by JoeMatt** [comment]
@claude can you make tickets and PRs with the changes to fix what we can as-is without updating the core yet. sounds like we have some bugs in some options or missing options.

also, what's the work tickets for updating mupen?

this one is complicated due to the plugins too, would be cool to have it work with xcodegen or spm first, to dynamically create the xcode compatible build, then maybe we can more easily update the core and plugins?
